static/app/utils/integrationUtil.tsx

```diff
@@ -228,3 +229,14 @@ export const getAlertText = (integrations?: Integration[]): string | undefined =
         'Update to the latest version of our Slack app to get access to personal and team notifications.'
       );
 };
+
+export const getExternalActorEndpointDetails = (
```

Comment:
This helper function will let us distinguish between sending to `{endpoint}` or `{endpoint}/{id}/` along with `PUT` vs `POST`. Even though in practice, baseEndpoint is always provided as `getBaseFormEndpoint(mapping)` we don't accept the function `getBaseFormEndpoint` as a param to keep this method pure.